class ContactException(Exception):
    pass


class ContactAlreadyExists(ContactException):
    pass
